tamars_cookies.md

```diff
@@ -0,0 +1,42 @@
+# Ingredients
+
+
+
```

Comment:
Can you remove extra empty lines? There should just be a single one in between each sections/titles/etc...